utils/record.js

```diff
@@ -0,0 +1,22 @@
+const { ValidationError } = require("./customError");
+var mongoose = require("mongoose");
+async function findRecordById(req, Model) {
+  let _Id = req.params.id;
+  let record = await Model.findById(_Id);

```

Comment:
Use const by default, only if rebinding is required use let. Update for both _Id & record